src/shared/keybindings.js

```diff
@@ -0,0 +1,28 @@
+import Mousetrap from "mousetrap";
+import tasks from "../editor/user-tasks/task-definitions";
```

Comment:
sorry @algentile, i should have been clearer in describing the objective -- regarding importing files between the folders `src/shared`, `src/editor`, and `src/eval-frame` we want to make sure that:

1.  no file in `src/editor` imports anything in `src/eval-frame`
2.  no file in `src/eval-frame` imports anything in `src/editor`
3.  no file in `src/shared` imports anything in `src/editor` or  `src/eval-frame`
4.  but files in  `src/editor` and  `src/eval-frame` may import from `src/shared`

you can think of  `src/editor` and  `src/eval-frame` as separate apps, so this import hygiene prevents us from accidentally pulling in a web of imports from one of the apps that will not be needed (or could even cause problems) in the other app.

i'd recommend putting the `initializeDefaultKeybindings` function back where it was in `src/editor`, and adding an additional file somewhere in `src/shared` that is just responsible for binding delete and backspace and handling/preventing back nav

does this make sense?

cc @wlach

Reply:
Hey @bcolloran @wlach, 

I moved keybindings back to initialization in editor and exported the intercept function from shared. The change should be redeployed in heroku.